.github/workflows/test.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - master
+    branches-ignore:
+      - "dependabot/**"
```

Comment:
> * skip dependabot branches

Could you explain this? Does this cause the tests to not be run on dependabot PR's?

Reply:
It only skip the branch builds since dependabot always opens a PR so it's duplicate runs :)

Reply:
BTW something must be wrong with this change because CI doesn't seem to be firing. I'll try to figure out why, later, and ping you.

Reply:
Okay, thanks! Should I merge or wait until you figure that out?

Reply:
Yup :) https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#excluding-branches-and-tags

I'll flip the check and see how it goes

Reply:
@raineorshine should be good to go now. Feel free to change the test to exclude the Unicode character :)

Reply:
Not sure if that was "yup" to merge or "yup" to wait, so I guess I'll wait! Thanks

Reply:
It's ready now :)